src/object.d

```diff
@@ -3241,6 +3241,7 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+// Compare floating point numbers for ordering (one instantiation per width).
```

Comment:
I'd suggest enforcing "one instantiation per width" using e.g. `static assert(is(Unqual!F == F));`